vms/platformvm/state/state.go

```diff
@@ -125,7 +125,9 @@ type State interface {
 	SetLastAccepted(blkID ids.ID)
 
 	GetStatelessBlock(blockID ids.ID) (blocks.Block, choices.Status, error)
-	AddStatelessBlock(block blocks.Block, status choices.Status)
+
+	// Invariant: [block] is an accepted block.
+	AddStatelessBlock(block blocks.Block)
```

Comment:
nit: should we rename this AddStatelessAcceptedBlock for clarify? Maybe it's too long of name

Reply:
I feel like this change would be reverted when we remove status from `GetStatelessBlock`